frontend/packages/dev-console/integration-tests/support/pages/add-flow/add-page.ts

```diff
@@ -10,61 +10,61 @@ export const addPage = {
     switch (card) {
       case 'Git':
       case addOptions.Git:
-        cy.byLegacyTestID('odc-add-card-item-import-from-git').click();
+        cy.byTestID('item import-from-git').click();
```

Comment:
Based on the current DOM structure, `data-test-id="odc-add-card-item-import-from-git"`. 

To use byTestID, data-* attribute needs to change as `data-test="odc-item import-from-git"` Have we updated this as part of any PR?

And also currently this attribute is present on div element. instead it should be on link to use "click" action

Reply:
Hey @makambalaji, I tested this by running (some) cypress tests incl. some which selects an "add card option" (or action, or item..) and they works fine. See attached screenshot:

![Screenshot from 2021-05-31 15-34-20](https://user-images.githubusercontent.com/139310/120216312-27b38b80-c237-11eb-972e-67c82ffaf5ce.png)

But, as we discussed earlier, I changed the structure of the add page card item to remove the `div` element between the `ul` and `li` element. /cc @nemesis09 Please see latest change https://github.com/openshift/console/compare/bee76ef4ab041e0ab0ac77a766c385f88567a130..2094faa4f6b0beca8b34ca2be6aa8c53c8b1f6f2

Before:
![Screenshot from 2021-05-31 17-32-57](https://user-images.githubusercontent.com/139310/120216903-e1126100-c237-11eb-80e8-56d88b7d949a.png)

After:
![Screenshot from 2021-05-31 17-32-26](https://user-images.githubusercontent.com/139310/120216897-deb00700-c237-11eb-9c91-e410276bebfb.png)

So, this does not not exactly address your issue. But because the Add Card works with an `onClick` element, I expect all works fine for the cypress tests with this change.